spiketools/utils/check_spikes_in_ms.py

```diff
@@ -0,0 +1,24 @@
+"""Spike related checking functions."""
+
+import numpy as np
+import warnings
```

Comment:
Small organizational comment: I tend to group imports like this:
```
standard-library

3rdparty-modules

local-module
```

This, I think, makes it a bit easier to quickly look and check the dependencies. 
So, for this example, this would look like:
```
import warnings

import numpy as np
```